website/content/en/docs/building-operators/ansible/installation.md

```diff
@@ -9,16 +9,22 @@ weight: 1
 Follow the steps in the [installation guide][install-guide] to learn how to install the `operator-sdk` CLI tool.
 
```

Comment:
we could maybe address in a follow up: 

I think we still missing add some kind of recommendation suggestions over the tool and deps. That should be recommended to upgrade the tool/project and use the deps scaffolds for each version to avoid issues. For example, by updating go/v2 with controller-gen 4.1 I would face issues, right?

So, my suggestion would we replicate the https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/quick-start.md#versions-and-supportability to sdk docs as well
+
Add a recommendation/steps over the best approach to update/upgrade the projects by following the changes proposed by each sdk tool version. we already answered and explained that many times in the channels but I think we have not it in the docs.

Reply:
>That should be recommended to upgrade the tool/project and use the deps scaffolds for each version to avoid issues. For example, by updating go/v2 with controller-gen 4.1 I would face issues, right?

Once an operator project is created, a user is free to update their project as they see fit, but operator-sdk doesn't guarantee that those updates will work _unless_ they are specified in docs/migration guides. I'd rather not be in the business of saying what users can't upgrade to. It's up to them to check compatibility of a specific tool/dep in that case.

Reply:
>  but operator-sdk doesn't guarantee that those updates will work unless they are specified in docs/migration guides

This is the point. we do not explain how we think is the best approach to keep the projects maintained and etc and why.  

>  I'd rather not be in the business of saying what users can't upgrade to. It's up to them to check the compatibility of a specific tool/dep in that case.

I agree that they are always able to do all that they want. However, what would be the best approach to avoid issues? how can we suggest they follow up on this process?

Reply:
>This is the point. we do not explain how we think is the best approach to keep the projects maintained and etc and why

We do, in the migration guides.

>However, what would be the best approach to avoid issues? what are the steps/workflow that we can recommend?

Follow the migration guides.